src/XMakeTasks/BootstrapperUtil/Util.cs

```diff
@@ -14,9 +14,7 @@ namespace Microsoft.Build.Tasks.Deployment.Bootstrapper
     internal static class Util
     {
         private const string BOOTSTRAPPER_REGISTRY_PATH_BASE = "Software\\Microsoft\\GenericBootstrapper\\";
-        private const string BOOTSTRAPPER_WOW64_REGISTRY_PATH_BASE = "Software\\Wow6432Node\\Microsoft\\GenericBootstrapper\\";
```

Comment:
I suspect that this is here for the case when 64-bit command-line MSBuild is run even though VS is 32-bit installed. Does that scenario no longer make sense?